### ISSUE TITLE
Unresolved reference

### ISSUE THREAD

**Issue by hafizhlf** [odoo-ide/vscode-odoo#62]
**Issue Description**:
I have been experiencing unresolved references in my Odoo project while working with VSCode. 

**Expected Behavior**:
I expect the references to be resolved correctly based on the project's configuration.

**Actual Behavior**:
Unresolved references are present, making it challenging to navigate and understand the code effectively.

**Environment**:
- Odoo Version: 17.0
- VSCode Version: 1.84.2
- Operating System: Windows 11

**Screenshots/Code Snippets**:
![image](https://github.com/odoo-ide/vscode-odoo/assets/15645495/828693cd-bcba-4040-9f8c-fdc8d541a443)

I would appreciate any guidance or assistance in resolving this matter. Thank you for your time and support.

**Comment by trinhanhngoc** [comment]
Hello @hafizhlf ,

This problem may be due to missing module depends in the manifest. If your module already depended on the `project` module, then the problem may be due to your project structure.

**Comment by hafizhlf** [comment]
I appreciate your assistance and guidance. Upon revisiting the issue and carefully following your suggestions, I have identified the root cause. The problem was indeed related to missing `project `information in the manifest file.

I have now added the necessary details to the manifest, and the unresolved references have been successfully resolved. I want to express my gratitude for pointing me in the right direction.
